content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,25 +9,81 @@ weight: 20
 ---
 
 <!-- overview -->
-This page describes common concepts in the Kubernetes API.
+The Kubernetes API is a resource-based (RESTful) programmatic interface
+provided via HTTP. It supports retrieving, creating, updating, and deleting
+primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE,
+GET).
 
+The API includes additional subresources for many objects that allow
+fine grained authorization (such as a separating viewing details for a Pod from
+retrieving its logs), and can accept and serve those resources in different
+representations for convenience or efficiency.
+
+Kubernetes supports efficient change notifications on resources via *watches*.
+Kubernetes also provides consistent list operations so that API clients can
+effectively cache, track, and synchronize the state of resources.
+
+You can view the [API reference](/docs/reference/kubernetes-api/) online,
+or read on to learn about the API in general.
 
 <!-- body -->
-The Kubernetes API is a resource-based (RESTful) programmatic interface provided via HTTP. It supports retrieving, creating,
-updating, and deleting primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE, GET), includes additional subresources for many objects that allow fine grained authorization (such as binding a pod to a node), and can accept and serve those resources in different representations for convenience or efficiency. It also supports efficient change notifications on resources via "watches" and consistent lists to allow other components to effectively cache and synchronize the state of resources.
+## Kubernetes API terminology {#standard-api-terminology}
+
+Kubernetes generally leverages standard RESTful terminology to describe the
+API concepts:
+
+* A *resource type* is the name used in the URL (`pods`, `namespaces`, `services`)
+* All resource types have a concrete representation in JSON (their object schema) which is called a *kind*
+* A list of instances of a resource type is known as a *collection*
+* A single instance of the resource type is called a *resource*
```

Comment:
```suggestion
* A single instance of the resource type is called an *object*
```

Reply:
(@deads2k should look at this too, he might disagree with me)

Reply:
I reworded to explain that most resources are also objects.

Reply:
IMO "resource" actually means what you're calling "resource type". It does not mean a specific object of that type.

Reply:
If we say we're following REST terminology: https://stackoverflow.com/a/10883810 has a relevant quote.

*However* I have seen a lot of Kubernetes contributors implying that, eg `.../deployments/` is a resource - and I've even done that myself. For this page I wanted to be more precise.

Reply:
Unfortunately (?) in k8s land, "resource" has come to be defined by opposition to "kind", the serialization format.

Reply:
Can we save that for a follow-up PR? _This_ PR isn't introducing a change here - see https://kubernetes.io/docs/reference/using-api/api-concepts/#standard-api-terminology for the live page.

Reply:
that's fine.